lib/config.js

```diff
@@ -110,12 +111,54 @@ module.exports = function (cfg) {
   }
 
   // --------------------------------------------------------------------------
-  // Settings: Available Selenium-ish options.
+  // Settings: Extract current desired setting path
   // --------------------------------------------------------------------------
-  if (!cfg.settings.default) {
-    cfg.settings.default = GLOBAL_DEFAULT;
+  var settingsPath = ROWDY_SETTINGS.replace(QUOTES_RE, "").split(".");
+  settingsPath = _.chain(settingsPath)
+    .map(function (v) { return v.trim(); })
+    .filter(_.identity)
+    .value();
+
+  // --------------------------------------------------------------------------
+  // Browser setting extensions (Guacamole)
+  // --------------------------------------------------------------------------
+  if (cfg.options.guacamole) {
+    // Lazy-require.
+    var guacamole = require("guacamole");
+
+    // SauceLabs.
+    if (settingsPath[0] === "sauceLabs") {
+      //
+      // TODO: Fix this path
```

Comment:
@Maciek416 -- OK. I propose:

```js
options: {
  guacamole: {
    enabled: true,
    shrinkwrap: path.join(__dirname, "guacamole-shrinkwrap.json")
  }
}
```

We'll push it down to consumers to override...